src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/ModellingGroupController.kt

```diff
@@ -64,6 +64,38 @@ open class ModellingGroupController(context: ControllerContext)
         return repo.getModellingGroupDetails(groupId)
     }
 
+    fun getModelRunParameterSets(context: ActionContext, estimateRepository: BurdenEstimateRepository): List<ModelRunParameterSet>
+    {
+        val touchstoneId = context.params(":touchstone-id")
+        val groupId = context.params(":group-id")
+        checkValidPermissions(groupId, touchstoneId, context, estimateRepository)
+        return estimateRepository.getModelRunParameterSets(groupId, touchstoneId)
+    }
+
+    private fun checkValidPermissions(
+            groupId: String,
+            touchstoneId: String,
+            context: ActionContext,
+            estimateRepository: BurdenEstimateRepository,
+            readEstimatesRequired: Boolean = false
+    )
+    {
+
+        val touchstones = estimateRepository.touchstoneRepository.touchstones
```

Comment:
Since a lot of this function is shared with `GroupBurdenEstimatesController`, maybe factor out to a common function in `CommonSecurityRules.kt`?